### ISSUE TITLE
Update for puppet 4.5.2 (agent 1.5.2)

### ISSUE THREAD

**Issue by robinbowes** [petems/puppet-install-shell#60]
Is there any change you can update the agent install script to support version 4.5.2 (which is puppet-agent 1.5.2) ?

Thanks,

R.

**Comment by robinbowes** [comment]
I actually ran into this when using the vagrant-puppet-install plugin. Is this script embedded in that plugin. or pulled in on install, or something?

**Comment by petems** [comment]
Yep, the plugin installs puppet using this script. You can also define your own script to use if this script is out of date:

```ruby
config.puppet_install.install_url = 'http://acme.com/install.sh'
```

**Comment by gesinn-it** [comment]
I get

```
"E: Version '1.5.2' for 'puppet-agent' was not found" with
Version: 4.5.1
Platform: debian
Platform Version: 8.5
Machine: x86_64
OS: Linux
```

and

```
E: Version '1.4.2' for 'puppet-agent' was not found
Version: 4.4.2
Platform: debian
Platform Version: 8.5
Machine: x86_64
OS: Linux
```

Is this related?

**Comment by petems** [comment]
@gesinn-it Hmm, interesting. That should've been caught by the debian acceptance test...

Let me look into that

**Comment by petems** [comment]
This should be fixed by #93
